diff --git a/apple/scripts/build-xcframework.sh b/apple/scripts/build-xcframework.sh
index fbb40943a219cc9b63190d55e981ee300feafbcc..c9824d922a7a5c0b227a2f0771b01f19fc0c80b0 100755
--- a/apple/scripts/build-xcframework.sh
+++ b/apple/scripts/build-xcframework.sh
@@ -213,7 +213,6 @@ build_slice() {
     -destination "generic/platform=${destination}" \
     -derivedDataPath "$DERIVED_DATA_PATH" \
     -configuration "$CONFIGURATION" \
-    -quiet \
     -disableAutomaticPackageResolution \
     -skipPackagePluginValidation \
     -skipMacroValidation \
